File: JavaScript/task2.js
//@ts-check
import {readFileSync} from 'fs';

var body  = readFileSync("TestFiles/test.json", "utf8");
const data = JSON.parse(body);

let arrayOfLists = data.lists;

arrayOfLists.forEach(element => {
    console.log("List id: " + element.id);
    console.log("List name: " + element.name)
})

File: Python/Task2.py
import json 

with open('testFile.json', 'r') as test_file:
    data = json.load(test_file)

    lists = data['lists']
 
    for i in range (len(lists)):
        print ("List ID:" + str(data['lists'][i]['id']))    
        print("List name:" + str(data['lists'][i]['name']))